app/views/dashboard/_buyer.html.erb

```diff
@@ -1,15 +1,45 @@
-<h1 style="margin-top: 120px; text-align:center;">Purchased Properties</h1>
 <% props = Property.where(buyer_id: current_user.id) %>
 
+<h1 style="margin-top: 130px; text-align:center;">Purchased Properties</h1>
+
 <section class="property-grid grid">
-      <div class="container">
-        <div class="row">
-          <div class="col-sm-12 justify-content-between">
-          <% props.each do |property| %>
-            <%= link_to property, class: "link-a" do  %>
-              <%= render property %>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+  <div class="container">
+    <div class="row">
+      <% props.each do |property| %>
+        <% seller = User.find(property.user_id) %>
+        <div class="box">
+          <div class="row">
+            <div class="col-sm-12 col-md-6" style="float:right;" >
+              <div class="testimonial-img">
+                <%= link_to property, class: "link-a" do  %>
+                  <%= render property %>
+                <% end %>
+              </div>
+            </div>
+            <div class="col-sm-12 col-md-6" style="float:left;">
+              <div class="testimonial-author-box text-center">
+                <h2>Property Details:</h2>
```

Comment:
manage these PROPERTY-DETAILS and SELLER-DETAILS into a separate page.

Reply:
- Made a partial for this